kernel/indtypes.ml

```diff
@@ -174,10 +164,13 @@ let infer_constructor_packet env_ar_par params lc =
   let lc'' = Array.map (fun j -> Term.it_mkProd_or_LetIn j.utj_val params) jlc in
   (* compute the max of the sorts of the products of the constructors types *)
   let levels = List.map (infos_and_sort env_ar_par) lc in
-  let isunit = is_unit levels in
+  (* An inductive definition is a "unit" if it has only one constructor
+     and that all arguments expected by this constructor are
+     logical, this is the case for equality, conjunction of logical properties
+  *)
```

Comment:
It’s weird to move the comment if you remove the code

Reply:
Removed while rebasing.